src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -139,9 +142,15 @@ const LoadingBarContainer = styled.div`
 const LogoContainer = styled.div`
   display: flex;
   align-items: flex-end;
-  margin-bottom: ${size.s};
 `;
 
+const SpaceBetween = styled.div`
+  margin-bottom: ${size.s};
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: flex-end;
+`;
```

Comment:
I don't think we need this styling; I think you can just slap a `gap` property on `LoadingBarContainer` to achieve the spacing

Reply:
Nice one! CSS PRO